README.md

```diff
@@ -1,2 +1,157 @@
-# Jester
-Jester - the Jolie rEST routER
+# Jester (Jolie 1.6.0 Beta or greater required)
+Jester - the Jolie rEST routER (and related tools)
+
+# tools/jolie2rest (Jolie 1.6.0 Beta or greater required)
+This tool allows for the creation of:
+- a declaration for the REST router in order to automatically publish an existing jolie microservice into the router
+- a Swagger interface which reports all the API published in the router
+
+Usage: jolie jolie2rest.ol router_host [swagger_enable=true|false] [easy_interface=true|false]
+Ex:
+```
+jolie jolie2rest.ol localhost:8080 swagger_enable=true easy_interface=false
+```
+
+It is worth noting that the parameter router_host identifies the location where the router is listening for, it does not represent the location of the target service.
+The tool will get the lits of the service to be analyzed from the file
+```
+service_list.txt
+```
+
+Each line of the file service_list.txt represents a service to be analyzed, in particular the line syntax is: path of the ol file of the service , name of the input port to be analyzed
+EX:
+```
+./demo/demo.ol, DEMO
+```
+
+It is important to highlight that a microservice can have more than one inputPort with different operations declared in it. If more inputPorts must be analyzed, then a line for each of them must be reported into the file service_list.txt.
+
+# REST annotations on the jolie interface
+In order to allow for the publishing of a jolie operation in a REST modality it is necessary to annotate each operation of the interfaces to be published as it follows:
+
+```
+/**! @Rest: method=[post|get|put|delete] [,template=...]; */
```

Comment:
Yes, let's do that for now... It's just that I don't like this situation where we have multiple ways of writing the same thing. Ideally, we would drop /**! in the future and I'd like not to encourage people to use it much further. But it's been used so much already that we'd need to provide an automatic script to port the code, so I guess this doesn't change much.